content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 ---
 id: hello-world
-title: Hello World
+title: Bonjour monde
```

Comment:
```suggestion
title: Bonjour, monde !
```

Le texte utilisé *in fine* est généralement celui-là, et comme ce sera en titre un peu partout, autant être péchus !